### PR TITLE
Added support for psu:InstPol

### PR DESCRIPTION
**Pull request by mfr-6**
Hi,

This PR is created to add support for CRUD operations for psu:InstPol which is missing. This is one of the resources I'm missing so for now I need to use "aci_rest" resource to have this configured. Because of that - this is not addressed under the "Issues" tab here. (Should I create an Issue describing a resource I want to implement before starting working on it?)

Please carry out the review, thank you.

Mateusz

### CODE REVIEW

File: client/psuInstPol_service.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	return sm.DeleteByDn(dn, models.PsuInstPolClassName)
+}
+
+func (sm *ServiceManager) UpdatePowerSupplyRedundancyPolicy(name string, description string, psuInstPolAttr models.PsuInstPolAttributes) (*models.PsuInstPol, error) {
+	rn := fmt.Sprintf(models.RnPsuInstPol, name)
+	psuInstPol := models.NewPowerSupplyRedundancyPolicy(rn, description, psuInstPolAttr)
+	psuInstPol.Status = "modified"
+	err := sm.Save(psuInstPol)
+	return psuInstPol, err
+}
```
Comment (akinross) [inline client/psuInstPol_service.go:37]: Is there a reason skip the List function?

Comment (mfr-6) [reply client/psuInstPol_service.go:37]: I missed that, my fault. I've just updated PR with additional commit

**Review by akinross — APPROVED** [review]
LGTM

**Review by sajagana — APPROVED** [review]
LGTM!

**Comment by mfr-6** [comment]
@lhercot since there are two approvals in place - I'd like to kindly ask you for a merge, thank you.

**Review by lhercot — APPROVED** [review]
LGTM

**Comment by lhercot** [comment]
Thanks for the contribution. I have created https://github.com/ciscoecosystem/aci-go-client/releases/tag/v2.16.0 that include your changes.

**Pull request merged**